eng/scripts/Approve-Eng-Common-Sync-PRs.ps1

```diff
@@ -16,66 +16,69 @@ if ($LASTEXITCODE -ne 0) {
   exit 1
 }
 
-$ghloggedInUser = (gh api user -q .login)
-$engCommonToolsBranch = gh pr view $engCommonSyncPRNumber -R Azure/azure-sdk-tools --json "headRefName" --jq ".headRefName"
+$ToolsRepo = "Azure/azure-sdk-tools"
 
-if (!$engCommonToolsBranch) {
-  Write-Error "Didn't find branch for PR $engCommonSyncPRNumber in Azure/azure-sdk-tools"
-  exit 1
+$ghloggedInUser = (gh api user -q .login)
```

Comment:
Maybe we should set `$PSNativeCommandUseErrorActionPreference = $true` along with error action preference stop at the top level?

Reply:
If we can't get logged in user do we not want to fail?

Reply:
Maybe :)

I can try setting that with my next common change to ensure that the error codes work as expected.